debian/rules

```diff
@@ -2,32 +2,59 @@
 
 DH_VERBOSE = 1
 
-DPKG_EXPORT_BUILDFLAGS = 1
-include /usr/share/dpkg/default.mk
-
+# We want package name to be named snuffleupagus, not php-snuffleupagus
```

Comment:
I'm ok with a package named `php-snuffleupagus` if this simplifies the packaging process.

Reply:
Making snuffleupagus a PECL package and calling it php-snuffleupagus would greatly simplifies the packaging because debhelper dh-php was made to compile PECL packages.

To make a PECL package we need to generate a package.xml https://pear.php.net/manual/en/pyrus.commands.make.php

But I failed to generate package.xml automatically and pyrus doesn't seem to be maintained anymore https://pear2.php.net/PEAR2_Pyrus

Reply:
What do you mean by "fail"? Please provide details.

If it's Snuffleupagus' fault, please open an issue, so we can fix it on our side.